### PR TITLE
raftstore: add metrics.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -825,6 +834,7 @@ impl Peer {
 
         match change_type {
             raftpb::ConfChangeType::AddNode => {
+                metric_incr!("raftstore.AddPeer");
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: The style Add/RemovePeer is not same as others.

File: src/raftstore/store/peer.rs [hunk 2]
```diff
@@ -362,6 +363,8 @@ impl Peer {
         } else {
             self.pending_cmds.append_normal(cmd);
         }
+
+        metric_incr!("raftstore.propose.success");
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: I think ok is short and easy to write.

Comment (ngaut) [inline src/raftstore/store/peer.rs]: Usually we use success/fail.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**